apollo-router/src/executable.rs

```diff
@@ -632,9 +632,21 @@ impl Executable {
             tracing::info!("Custom plugins are present. To see log messages from your plugins you must configure `RUST_LOG` or `APOLLO_ROUTER_LOG` environment variables. See the Router logging documentation for more details");
         }
 
+        let uplink_config = opt.uplink_config().ok();
+        if uplink_config
+            .clone()
+            .unwrap_or_default()
+            .endpoints
+            .unwrap_or_default()
+            .url_count()
+            == 1
+        {
+            tracing::warn!("Only a single uplink endpoint is configured. We recommend specifying at least two endpoints so that a fallback exists.");
```

Comment:
IIUC this is pretty early in Router startup and isn't decorated by any other context. Do we think this message is clear and actionable enough or would it help to link to relevant docs or explicitly name the configuration points (`--apollo-uplink-endpoints`/`APOLLO_UPLINK_ENDPOINTS`? It also looks like the docs mostly refer to this as "Apollo Uplink" rather than "uplink", though I don't know if other in-app text does.